attester/src/lib.rs

```diff
@@ -12,26 +12,31 @@ pub mod sample;
 
 #[cfg(feature = "az-snp-vtpm-attester")]
 pub mod az_snp_vtpm;
+
 #[cfg(feature = "tdx-attester")]
 pub mod tdx;
 
 #[cfg(feature = "occlum-attester")]
 pub mod sgx_occlum;
 
+#[cfg(feature = "snp-attester")]
+pub mod snp;
+
 /// The supported TEE types:
 /// - Tdx: TDX TEE.
 /// - SgxOcclum: SGX TEE with Occlum Libos.
 /// - AzSnpVtpm: SEV-SNP TEE for Azure CVMs.
-/// - Sevsnp: SEV-SNP TEE.
+/// - Snp: SEV-SNP TEE.
 /// - Sample: A dummy TEE that used to test/demo the KBC functionalities.
 #[derive(Debug, EnumString, Display)]
 #[strum(ascii_case_insensitive, serialize_all = "lowercase")]
 pub enum Tee {
     Tdx,
     #[strum(serialize = "sgx")]
     SgxOcclum,
-    Sevsnp,
     AzSnpVtpm,
+    Sgx,
```

Comment:
there's an `SgxOcclum` in line 36; do we need this as well?

Reply:
IMO, we do not need this.